.github/workflows/releaseutils.yml

```diff
@@ -0,0 +1,30 @@
+name: Add sbom
+
+on:
+  workflow_dispatch:
+  workflow_run:
+        workflows: [test-latest-release]
+        types: [completed]
+
+permissions: read-all

```

Comment:
Does this thwart the upload step?

Reply:
BTW, if you want to grant only the needed write privileges, I think https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents identifies what is needed for writing "assets".

Reply:
BTW, if the SBOM is less than 1MB then https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/passing-information-between-jobs shows how you can link-up differently-privileged jobs in a workflow.

Reply:
answered in the other review, and yes I'll use granular write for the specific job if required.